### PR TITLE
Adding "beta" banners to relevant error report sections

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #_

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.
> *Demo GIF(s) and screenshots for testing procedure*

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [ ] [**_insert ACs here_**]
+ [ ] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [ ] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [ ] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [ ] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

**Review by Smithh-Co — APPROVED** [review]

File: tdrs-backend/tdpservice/parsers/views.py [hunk 1]
```diff
@@ -50,6 +50,9 @@ def _get_xls_serialized_file(self, data):
                           'row_number',
                           'column_number']
 
+        # write beta banner
+        worksheet.write(row, col, "BETA TESTING - PLEASE DO NOT USE - REFER TO LEGACY TDRS EMAILS")
```
Comment (reitermb) [inline tdrs-backend/tdpservice/parsers/views.py]:
```suggestion
        worksheet.write(row, col, "Error reporting in TDP is still in development. We'll be in touch when it's ready to use! For now please refer to the reports you receive via email")
```

File: tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const SectionSubmissionHistory = ({ section, label, files }) => {
                 <th>Submitted On</th>
                 <th>Submitted By</th>
                 <th>File Name</th>
-                <th>Error Reports</th>
+                <th>**IN BETA** Error Reports</th>
```
Comment (reitermb) [inline tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx]:
```suggestion
                <th>Error Reports (In development)</th>
```

File: tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.test.js [hunk 3]
```diff
@@ -241,7 +241,7 @@ describe('SubmissionHistory', () => {
     expect(screen.queryByText('test5.txt')).not.toBeInTheDocument()
     expect(screen.queryByText('test6.txt')).toBeInTheDocument()
 
-    expect(screen.queryByText('Error Reports')).toBeInTheDocument()
+    expect(screen.queryByText('**IN BETA** Error Reports')).toBeInTheDocument()
```
Comment (reitermb) [inline tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.test.js]:
```suggestion
    expect(screen.queryByText('Error Reports (In development)')).toBeInTheDocument()
```

**Review by reitermb — CHANGES_REQUESTED** [review]
All requested changes added as inline suggestions

File: tdrs-backend/tdpservice/parsers/views.py (hunk 1 above)
Comment (reitermb) [reply tdrs-backend/tdpservice/parsers/views.py]: In addition to the above copy tweak could we add a blank row between this message and the column headers for some added visual separation

File: tdrs-backend/tdpservice/parsers/views.py [hunk 4]
```diff
@@ -50,6 +50,9 @@ def _get_xls_serialized_file(self, data):
...
+        row, col = 1, 1
```
Comment (andrew-jameson) [inline tdrs-backend/tdpservice/parsers/views.py]:
```suggestion
        row, col = 2, 0
```

File: tdrs-backend/tdpservice/parsers/views.py [hunk 5]
```diff
@@ -50,6 +50,9 @@ def _get_xls_serialized_file(self, data):
                           'row_number',
                           'column_number']
 
+        # write beta banner
+        worksheet.write(row, col, "Error reporting in TDP is still in development. We'll be in touch when it's ready to use! For now please refer to the reports you receive via email")
```
Comment (andrew-jameson) [inline tdrs-backend/tdpservice/parsers/views.py:54]:
```suggestion
        worksheet.write(row, col, \
                                   "Error reporting in TDP is still in development. We'll be in touch when it's ready to use! \
                                    For now please refer to the reports you receive via email")
```

**Review by reitermb — APPROVED** [review]
Looks good! 🙌

**Comment by ADPennington** [comment]
@andrew-jameson circleci tests need a little help

**Comment by andrew-jameson** [comment]
> @andrew-jameson circleci tests need a little help

Yup, should be addressed now. The new text was too longer for linting.

**Review by ADPennington — APPROVED** [review]

**Pull request merged**